examples/server-side-rendering/package.json

```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "core-js": "^3.0.1",
+    "cross-env": "^7.0.0",
```

Comment:
devDependency please.

Reply:
The `start` script command also use `NODE_ENV`

Reply:
all scripts are "dev" scripts. Final users are going to use `/dist` + `deps`. Everything not required for the final users should be gone.